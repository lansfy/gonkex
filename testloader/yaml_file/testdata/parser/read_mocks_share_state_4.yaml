- name: share state MUST do nothing for this testcase ...
  method: GET
  path: /some/path1
  request: "requestbody"
  mocksParams:
    shareState: true
  mocks:
    someservice:
      strategy: nop

- name: ... because this testcase does not have shareState
  method: GET
  path: /some/path2
  request: "requestbody"
