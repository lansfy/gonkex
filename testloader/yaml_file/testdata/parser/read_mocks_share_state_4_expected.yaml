- GetName: share state MUST do nothing for this testcase ...
  GetMethod: GET
  Path: /some/path1
  GetRequest: requestbody
  GetFileName: testdata/parser/read_mocks_share_state_4.yaml
  FirstTestInFile: true
  ServiceMocks:
    someservice:
      strategy: nop

- GetName: ... because this testcase does not have shareState
  GetMethod: GET
  Path: /some/path2
  GetRequest: requestbody
  GetFileName: testdata/parser/read_mocks_share_state_4.yaml
  LastTestInFile: true
